### PR TITLE
[1577, 1575] Remove financial support, move fees to own page

### PR DESCRIPTION
**Pull request by elceebee**
### Context
After the analysis of free text fields, we do not need both the financial support and fee details fields. 
AND
We are moving the free text fields into their own page

[Trello ticket for removing financial support](https://trello.com/c/7lb2uHHF)
[Trello ticket for moving fees to their own page](https://trello.com/c/BgWXv7hr)

### Changes proposed in this pull request
- Removes opportunity for users to edit / enter data into the financial support text box (no migration, no removing data from the database)
- only show the financial support information conditionally on the summary page
- Content changes to (what is now) the fee details page
- Uk / international fees now on their own page
- No copy content available on either page

https://github.com/DFE-Digital/publish-teacher-training/assets/44073106/6c739f25-d485-420c-8eb7-159d680071cb

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [NA] Inform data insights team due to database changes

### CODE REVIEW

**Comment by elceebee** [comment]
[Publish: Cut down Course Length & Fees page - Remove financial_support](https://trello.com/c/7lb2uHHF/1577-publish-cut-down-course-length-fees-page-remove-financialsupport)

**Comment by elceebee** [comment]
[Publish: Cut down Course Length & Fees page - move Fees to its own page](https://trello.com/c/BgWXv7hr/1575-publish-cut-down-course-length-fees-page-move-fees-to-its-own-page)

**Review by tomas-stefano — APPROVED** [review]

File: app/forms/publish/course_fees_and_financial_support_form.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Publish
+  class CourseFeesAndFinancialSupportForm < BaseModelForm
+    include RecruitmentCycleHelper
+
+    alias course_enrichment model
+
+    include FundingTypeFormMethods
```
Comment (inulty-dfe) [inline app/forms/publish/course_fees_and_financial_support_form.rb:9]: Let's put the includes together

File: app/views/publish/courses/fees_and_financial_support/_guidance.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<%= t("publish.providers.fees_and_financial_support.edit.guidance_html") %>
+<details class="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      <%= t("publish.providers.fees_and_financial_support.edit.summary_link") %>
+    </span>
+  </summary>
+  <div class="govuk-!-margin-top-4" />
+  <%= render Find::Utility::AdviceComponent::View.new(title: "Financial support from the government") do %>
```
Comment (inulty-dfe) [inline app/views/publish/courses/fees_and_financial_support/_guidance.html.erb:9]: This has the namespace `Find` but we're using it in Publish views. Can we find a way to make this work?

Maybe remove it from the Find namespace into a shared one or something?

Comment (inulty-dfe) [reply app/views/publish/courses/fees_and_financial_support/_guidance.html.erb:9]: Same for all the components and views below

File: spec/features/publish/editing_fees_and_financial_support_spec.rb [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    when_i_visit_the_fees_and_financial_support_edit_page
+    then_i_see_the_new_fees_and_financial_support_information
+  end
+
+  scenario 'entering invalid data' do
+    given_i_am_authenticated_as_a_provider_user
+    and_there_is_a_course_i_want_to_edit
+    when_i_visit_the_fees_and_financial_support_edit_page
+    and_i_enter_invalid_data
```
Comment (inulty-dfe) [inline spec/features/publish/editing_fees_and_financial_support_spec.rb:22]:
```suggestion
    and_i_enter_too_many_words
```

**Review by inulty-dfe — CHANGES_REQUESTED** [review]
Nice work, very clean :100: 

Only real concern is mixing up of namespaces

**Review by dcyoung-dev — APPROVED** [review]

**Pull request merged**